### DISCUSSION TITLE
3.3v issue?

### DISCUSSION THREAD

**Discussion by Amahumahaba** [darthcloud/BlueRetro#96] [General]
Hi, I've put together the Bluetooth adapter with a single n64 extension cable direct to board, but when it isn't plugged into USB power it shuts off the n64 when I plug it in. When USB is attached, it doesn't do this.

Is this because I haven't connected the unused data pins to 3.3v? I hadn't seen anyone's video guides doing that part so I had avoided it so far.

Thanks.

**Comment by darthcloud** [comment]
It's not related to the unconnected data line for sure.

The thing is what I tested is the official boards, but they are sold out.

Not sure if it problem with those 3rd party ESP board or its specific to some N64.

**Reply by SteelBattalion** [reply]
I've built one and have one working on a third party ESP-WROOM-32 board. Where are you pulling 3.3v from? I am pulling mine from the chonky pad that the controller port is attached to. I haven't tested plugging a usb cable in while my system is powered on as I haven't needed to for firmware updates since it pulls power from the system
![IMG_20211130_023137_2](https://user-images.githubusercontent.com/21997826/144027590-0397f818-65a4-4722-a409-a273aa630106.jpg)
![IMG_20211130_023159_2](https://user-images.githubusercontent.com/21997826/144027592-43195aa9-0fc5-4f45-8f99-80ef8727fd30.jpg)
![IMG_20211130_023253_2](https://user-images.githubusercontent.com/21997826/144027593-ca3cda81-1786-41f8-8b4d-bcd432e5fd04.jpg)
![IMG_20211130_025234_2](https://user-images.githubusercontent.com/21997826/144027595-7c733145-dc39-4af7-b3ac-7373ff121ff5.jpg)
![IMG_20211130_023056_2](https://user-images.githubusercontent.com/21997826/144027596-7f331f13-cd88-4cc4-b198-ea083e9e10c8.jpg)
.

**Comment by persechini** [comment]
What is the purpose of connecting the unused data pins to 3.3V? It's just so that is stops searching for controllers after the last connected one is paired? If so, is there any distinction between a single controller adapter without the pins connected and a 4 controller adapter that is currently only connected to one controller?

**Reply by darthcloud** [reply]
The purpose is for the input to be stable in an idle state to not trigger infinite interupt that will crash the ESP.

Sometimes people are lucky and it work without.

Sometimes people don't follow instructions and wonder why it doesn't work.
